### ISSUE TITLE
sysupgrade and custom build requests failing

### ISSUE THREAD

**Issue by lorand-horvath** [openwrt/asu#961]
@aparcar @efahl ASU and firmware-selector fail to build for a few days now, it seems the sysupgrade server is down or its API is somehow broken.
Please see reports:
https://github.com/openwrt/openwrt/issues/15192#issuecomment-2331246644
https://forum.openwrt.org/t/the-openwrt-firmware-selector/81721/874
https://forum.openwrt.org/t/the-openwrt-firmware-selector/81721/885

**Comment by efahl** [comment]
Yeah, the front end API server there was completely dead for a while (503 errors fetching stuff from json/v1), then it came back.  Still dead with any build requests are just returning `Internal Server Error'.

**Comment by hainguyen-cmd** [comment]
Until now, ASU still have problem! Waiting for @aparcar @efahl !!

**Comment by efahl** [comment]
Sorry, not much I can do, I only work on the clients (and sometimes submit PRs to ASU).  Paul is the only one I know who can actually log into the servers.

**Comment by sqliuchang** [comment]
[https://sysupgrade.openwrt.org/api/v1/build](https://sysupgrade.openwrt.org/api/v1/build) is still not working.

**Comment by sqliuchang** [comment]
It works now.

**Comment by hainguyen-cmd** [comment]
Still got error:
`Downloading file:packages/Packages
Updated list of available packages in /builder/build_dir/target-mipsel_24kc_musl/root-ramips/../../../../builder/dl/imagebuilder
Downloading file:packages/Packages.sig
Signature check passed.
Collected errors:
 * opkg_install_cmd: Cannot install package luci-app-wireguard.
make[2]: *** [Makefile:220: package_install] Error 255
make[1]: *** [Makefile:161: _call_manifest] Error 2
make: *** [Makefile:322: manifest] Error 2`

**Comment by efahl** [comment]
> opkg_install_cmd: Cannot install package luci-app-wireguard.

That's a package issue, nothing to do with ASU server.  You can check on the download server to see if it's there, and try again later.

**Comment by hainguyen-cmd** [comment]
> > opkg_install_cmd: Cannot install package luci-app-wireguard.
> 
> That's a package issue, nothing to do with ASU server. You can check on the download server to see if it's there, and try again later.

Thanks so much! I have checked again. Everything work fine!
